server/manifest.go

```diff
@@ -123,7 +123,7 @@ func WriteManifest(name model.Name, config Layer, layers []Layer) error {
 	return json.NewEncoder(f).Encode(m)
 }
 
-func Manifests() (map[model.Name]*Manifest, error) {
+func Manifests(reportCorruption bool) (map[model.Name]*Manifest, error) {
```

Comment:
`reportCorruption` is quite accurate since a valid, non-JSON file can exist in this directory. I would invert it and name it `continueOnError`